src/smdba/postgresqlgate.py

```diff
@@ -155,9 +155,7 @@ def estimate(self):
         # No more than 1GB
         self.config['maintenance_work_mem'] = self.to_mb(self.br((mem / 0x10) > megabytes and megabytes or mem / 0x10))
 
-        self.config['checkpoint_segments'] = 8
         self.config['checkpoint_completion_target'] = '0.7'
-        self.config['wal_buffers'] = self.to_mb(0x200 * self.config['checkpoint_segments'])
```

Comment:
wal_buffers by default gets set to 3% of shared buffers while we currently are using a hardcoded value of 4MB always.